package-lock.json

```diff
@@ -12397,9 +12397,9 @@
       }
     },
     "node_modules/npm-package-arg": {
-      "version": "12.0.1",
-      "resolved": "https://registry.npmjs.org/npm-package-arg/-/npm-package-arg-12.0.1.tgz",
-      "integrity": "sha512-aDxjFfPV3Liw0WOBWlyZLMBqtbgbg03rmGvHDJa2Ttv7tIz+1oB5qWec4psCDFZcZi9b5XdGkPdQiJxOPzvQRQ==",
+      "version": "12.0.2",
```

Comment:
There is as breaking change causing a CI failure in 12.0.2. Not sure what that is about or how we go about resolving that here :)

Reply:
This is an annoying bug that only affects npm itself because of how it rebuilds its bundled deps.  It brings in updates during the process.  Once we get a deps update PR landed on the cli this will go away (until the next time).